nerfstudio/scripts/viewer/sync_viser_message_defs.py

```diff
@@ -34,7 +34,7 @@ def main() -> None:
     # - nerfstudio/scripts/viewer/
     # - nerfstudio/scripts/
     # - nerfstudio/
-    target_path = pathlib.Path(__file__).absolute().parent.parent.parent / pathlib.Path(
+    target_path = pathlib.Path(__file__).absolute().parent.parent.parent.parent / pathlib.Path(
```

Comment:
explain? (If this is updated it seems like the comment above should be as well?)

Reply:
@tancik and I changed this after ns-dev-sync failed because the path of the scripts/ directory changed

Reply:
The comment still holds, I think it was wrong originally.

Reply:
It looks to me like we should drop one `.parent`, and remove the `nerfstudio/` in the string on the next line

Reply:
fixed